axe-violations/f-checkout-a11y-violations.csv

```diff
@@ -0,0 +1 @@
http://localhost:8080/iframe.html?id=components-organisms--checkout-component&viewMode=story,select-name,critical,https://dequeuniversity.com/rules/axe/4.1/select-name?application=axeAPI,<select id="time-selection" data-test-id="formfield-order-time-dropdown-select" name="order-time" class="FormDropdown_c-formDropdown-select_ROvjN">,aria-label attribute does not exist or is empty--aria-labelledby attribute does not exist- references elements that do not exist or references elements that are empty--Form element does not have an implicit (wrapped) <label>--Form element does not have an explicit <label>--Element has no title attribute--Element's default semantics were not overridden with role="none" or role="presentation",#time-selection
http://localhost:8080/iframe.html?id=components-organisms--checkout-component&viewMode=story,select-name,critical,https://dequeuniversity.com/rules/axe/4.1/select-name?application=axeAPI,<select id="time-selection" data-test-id="formfield-order-time-dropdown-select" name="order-time" class="FormDropdown_c-formDropdown-select_ROvjN">,aria-label attribute does not exist or is empty--aria-labelledby attribute does not exist- references elements that do not exist or references elements that are empty--Form element does not have an implicit (wrapped) <label>--Form element does not have an explicit <label>--Element has no title attribute--Element's default semantics were not overridden with role="none" or role="presentation",#time-selection
http://localhost:8080/iframe.html?id=components-organisms--checkout-component&viewMode=story,select-name,critical,https://dequeuniversity.com/rules/axe/4.1/select-name?application=axeAPI,<select id="time-selection" data-test-id="formfield-order-time-dropdown-select" name="order-time" class="FormDropdown_c-formDropdown-select_ROvjN">,aria-label attribute does not exist or is empty--aria-labelledby attribute does not exist- references elements that do not exist or references elements that are empty--Form element does not have an implicit (wrapped) <label>--Form element does not have an explicit <label>--Element has no title attribute--Element's default semantics were not overridden with role="none" or role="presentation",#time-selection
http://localhost:8080/iframe.html?id=components-organisms--checkout-component&viewMode=story,select-name,critical,https://dequeuniversity.com/rules/axe/4.1/select-name?application=axeAPI,<select id="time-selection" data-test-id="formfield-order-time-dropdown-select" name="order-time" class="FormDropdown_c-formDropdown-select_ROvjN">,aria-label attribute does not exist or is empty--aria-labelledby attribute does not exist- references elements that do not exist or references elements that are empty--Form element does not have an implicit (wrapped) <label>--Form element does not have an explicit <label>--Element has no title attribute--Element's default semantics were not overridden with role="none" or role="presentation",#time-selection

```

Comment:
Should this have been checked in?

Reply:
You were too quick for me.  It shouldn't have been.

Reply:
Would be better to prevent adding these files in in the first place; could we add this directory to the `.gitignore`, either in this or another PR?

Reply:
Aye, @siggerzz and me noticed that.  I'll put it in as a separate PR though, to keep things nice and tidy.